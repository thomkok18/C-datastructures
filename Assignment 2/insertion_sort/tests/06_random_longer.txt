1
32
55
17
55
22
1
16
32
32
1
