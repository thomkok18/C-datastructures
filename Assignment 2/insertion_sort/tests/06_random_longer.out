1
1
1
16
17
22
32
32
32
55
55
